Gemfile

```diff
@@ -10,6 +10,7 @@ gem "carrierwave-i18n"
 gem "chronic"
 gem "dalli", "~> 2.7"
 gem "faraday"
+gem "fog-aws", "~> 3.5"
```

Comment:
It would be good to link to the specialist publisher PRs in the commit message itself.